client/src/views/workspaces/WorkspacesPage.vue

```diff
@@ -346,10 +356,16 @@ async function handleFileLink(fileLink: ParsecOrganizationFileLinkAddr): Promise
 }
 
 async function onDisplayStateChange(): Promise<void> {
-  await storageManager.storeComponentData<WorkspacesPageSavedData>(WORKSPACES_PAGE_DATA_KEY, { displayState: displayView.value });
+  await storageManager.storeComponentData<WorkspacesPageSavedDisplayState>(WORKSPACES_PAGE_DATA_KEY, { displayState: displayView.value });
 }
 
 async function refreshWorkspacesList(): Promise<void> {
+  const savedFavorites = await storageManager.retrieveComponentData<WorkspacesPageSavedFavorites>(WORKSPACES_PAGE_DATA_KEY);
```

Comment:
This list doesn't need to be retrieved everytime we refresh the workspaces.

But the list of favourites does need to be refreshed everytime a change occurs on it. This means we need a specific event to warn both the sidebar and the workspaces page that the list has been updated. See `eventDistributor`.